tests/bluetooth/tester/testcase.yaml

```diff
@@ -0,0 +1,5 @@
+tests:
+  bluetooth.general.tester:
+    platform_allow: qemu_x86
+    tags: bluetooth
+    harness: bluetooth
```

Comment:
Could you please add an additional target for building on nrf52840, just so the tests can be compiled as part of CI?

ie
```yml
  bluetooth.general.tester.build:
    build_only: true
    platform_allow: nrf52840dk_nrf52840
    tags: bluetooth ci_build
```